crates/stc_ts_types/src/lib.rs

```diff
@@ -281,7 +281,7 @@ impl Clone for Type {
 }
 
 #[cfg(target_pointer_width = "64")]
-assert_eq_size!(Type, [u8; 104]);
+assert_eq_size!(Type, [u8; 120]);
```

Comment:
Also, is this unevitable?

Reply:
If you want to add a value to CommonMetadata, you must.
It contains metadata about the values before they were assigned unstructured.